libs/core/src/graphql/get-fragment-names.spec.ts

```diff
@@ -0,0 +1,63 @@
+import { DocumentNode } from 'graphql';
+import gql from 'graphql-tag';
+import { daffGetFragmentNames } from './get-fragment-names';
+
+describe('Core | GraphQL | daffGetFragmentNames', () => {
```

Comment:
Can you provide an integration test of how this would be used to merge fragments into queries?

Reply:
done